splunk/common-files/Dockerfile

```diff
@@ -23,10 +23,14 @@ COPY splunk/common-files/make-minimal-exclude.py /tmp
 RUN python /tmp/make-minimal-exclude.py ${SPLUNK_BUILD_URL} > /tmp/splunk-minimal-exclude.list \
     && echo "Downloading Splunk and validating the checksum at: ${SPLUNK_BUILD_URL}" \
     && wget -qO /tmp/`basename ${SPLUNK_BUILD_URL}` ${SPLUNK_BUILD_URL} \
-    && wget -qO /tmp/splunk.tgz.sha512 ${SPLUNK_BUILD_URL}.sha512 \
+    && ((wget -qO /tmp/splunk.tgz.sha512 ${SPLUNK_BUILD_URL}.sha512 \
     && cd /tmp \
     && echo "$(cat /tmp/splunk.tgz.sha512)" | sha512sum --check  --status \
-    && rm /tmp/splunk.tgz.sha512 \
+    && rm /tmp/splunk.tgz.sha512 ) \
+    || (wget -qO /tmp/splunk.tgz.sha256 ${SPLUNK_BUILD_URL}.sha256 \
+    && cd /tmp \
+    && echo "$(cat /tmp/splunk.tgz.sha256) `basename ${SPLUNK_BUILD_URL}`" | sha256sum --check  --status \
+    && rm /tmp/splunk.tgz.sha256)) \
```

Comment:
this needs to be done since sha512 checksums are not available for splunkcloud artifactory builds.